docs/hole-finder.py

```
@@ -0,0 +1,67 @@
+from pymatgen.io.vasp.outputs import BSVasprun
+from pymatgen.electronic_structure.core import Spin
+import pandas as pd
+import warnings
+
+warnings.filterwarnings("ignore", message="No POTCAR file with matching TITEL fields")
+
+
+def hole_finder(vasprun_defect, bands_below=3, bands_above=3):
```

Ruff (0.8.2)

Missing docstring in public function

(D103)